src/fork_gc.c

```diff
@@ -844,6 +844,9 @@ static int recvCardvals(ForkGC *fgc, arrayof(CardinalityValue) *tgt, size_t *len
   if (*tgt) {
     rm_free(*tgt);
   }
+  if (*tgt) {
+    rm_free(*tgt);
+  }
```

Comment:
This file should not be in this PR, right?
(and this snippet is redundant)

Reply:
Yes, this was already merged to master, I'll remove it.